packages/formstr-app/src/containers/CreateFormNew/components/AIFormGeneratorModal/OllamaSettings.tsx

```diff
@@ -60,14 +60,15 @@ const OllamaSettings: React.FC<OllamaSettingsProps> = ({
           <Title level={5}>2. Start Ollama Server (macOS / Linux / Windows)</Title>
           <Paragraph>
             Use the command block below based on your platform to start the Ollama server:
+            Make sure to <Text strong>restart Ollama</Text> after setting these environment variables if it's already running.
           </Paragraph>
 
           <Row gutter={16}>
             <Col xs={24} md={12}>
               <Text strong>macOS / Linux / WSL</Text>
               <CodeBlock>
                 export OLLAMA_HOST=0.0.0.0:11434{'\n'}
-                export OLLAMA_ORIGINS='*'{'\n'}
+                export OLLAMA_ORIGINS='https://formstr.app,http://localhost:3000'{'\n'}
```

Comment:
should use window.location ,since we don't know where the app would be running from